.github/workflows/ui_publish_aap_chatbot.yml

```diff
@@ -0,0 +1,43 @@
+name: Chatbot - Build and Test
```

Comment:
Probably want to change this name (to avoid duplication with the Action that does the build and test and is called the same)